### PR TITLE
working icon dragging

### PR DESCRIPTION
**Pull request by tomermoshe**
added directive
added docs
updated changelog

### CODE REVIEW

File: demo/app/app.component.ts [hunk 1]
```diff
@@ -41,6 +43,11 @@ export class AppComponent implements AfterViewInit {
 
   ngAfterViewInit(): void {
     // example for getting the viewer by Id outside of the ac-map hierarchy
-    const viewer = this.viewersManager.getMap('main-map');
+    const map = this.mapsManagerService.getMap('main-map');
+    const viewer = map.getCesiumViewer();
+
+    const coordinateConverter = map.getCoordinateConverter();
+    this.draggableToMapService.init(coordinateConverter);
```
Comment (eitanfr) [inline demo/app/app.component.ts]: user needs to provide the coordinate converter?
maybe `draggableToMapService` will create his own `coordinateConverter` using this.mapsManagerService.getMap() default map, if the user has a number of maps he will be able to pass map-id.

File: demo/app/app.component.ts [hunk 2]
```diff
@@ -41,6 +43,11 @@ export class AppComponent implements AfterViewInit {
...
+    this.draggableToMapService.observable().subscribe(e => console.log(e));
```
Comment (eitanfr) [inline demo/app/app.component.ts]: rename from observable to dragUpdates or somthing like this

Comment (tomermoshe) [reply demo/app/app.component.ts]: why? its the exposed observable - not like there are any other observables

File: demo/app/app.component.ts (hunk 1 above)
Comment (tomermoshe) [reply demo/app/app.component.ts]: ok

File: demo/app/app.component.ts (hunk 2 above)
Comment (tomermoshe) [reply demo/app/app.component.ts]: done

**Pull request merged**